pennylane/plugins/default_qubit.py

```diff
@@ -58,6 +58,7 @@ class DefaultQubit(QubitDevice):
         "PauliX",
         "PauliY",
         "PauliZ",
+        "PauliRot",
```

Comment:
Going back to the previous comment, if the decomposition is more efficient, does it make more sense to _not_ list `"PauliRot"` here, to force the decomposition within default qubit?

Reply:
A side effect of this is that the decomposition will work with the metric tensor method, but the direct PauliRot gate won't. Same thing happens with `Rot`; if you want to use a circuit with `Rot` and calculate the metric tensor, you need to remove it from this list 😆

Reply:
I will do some benchmarks! 

But if you want to use the metric tensor you can also do this hack in your client side code: 
```
qml.PauliRot = qml.PauliRot._decomposition
```